experimental/python/README.md

```diff
@@ -0,0 +1,29 @@
+# databricks-bundles
+
+Python for Databricks Asset Bundles extends [Databricks Asset Bundles](https://docs.databricks.com/aws/en/dev-tools/bundles/) so that you can:
+- Define jobs and pipelines as Python code. These jobs can coexist with jobs defined in YAML.
+- Dynamically create jobs and pipelines using metadata.
+- Modify jobs and pipelines defined in YAML or Python during bundle deployment.
```

Comment:
WDYT?
```suggestion
- Modify jobs and pipelines defined in YAML or Python during bundle deployment at runtime.
```

Reply:
We don't want to say runtime, because it can be confused with job runtime that is not true. It was a common source of confusion until we made it very straightforward.